pkg/api/domains/domains.go

```diff
@@ -72,9 +80,13 @@ func (c *Client) Create(ctx context.Context, req *CreateRequest) (DomainResponse
 func (c *Client) Update(ctx context.Context, req *UpdateRequest) (DomainResponse, error) {
 	str := strconv.FormatInt(req.Id, 10)
 	request := c.apiClient.DomainsApi.UpdateDomain(ctx, str).UpdateDomainRequest(req.UpdateDomainRequest)
+	logger.Debug("request", zap.Any("request", request))
+
 	domainsResponse, httpResp, err := request.Execute()
+	logger.Debug("response", zap.Any("response struct", domainsResponse), zap.Any("response http", httpResp), zap.Error(err))
 
 	if err != nil {
+		logger.Error("Update request.Execute return error", zap.Error(err))
```

Comment:
Another detail:
I think also showing the HttpRes body is useful.... sometimes the err returned from the SDK is not descriptive enough.

Reply:
já vai ser retornado acima